src/settings.py

```diff
@@ -31,14 +32,14 @@ class Settings(BaseSettings):
     uvicorn_proxy_headers: bool = True
 
     # supabase settings
-    processor_username: str = 'processor@deadtrees.earth'
-    processor_password: str = 'processor'
+    processor_username: str = "processor@deadtrees.earth"
+    processor_password: str = "processor"
```

Comment:
can you please stop using double quotes in Python? :)

Reply:
try to disable the formatter